sdk/template/template/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/template",
-  "version": "0.2.0-preview.2",
+  "version": "1.0.0-preview.3",
```

Comment:
this feels like an odd version change. 0.2 -> 0.3 counts as a major version change for pre-release package, but preview suffixes allow breaking changes anyway. Can we just make the template 1.0.0 ? 

Since this package never publishes, I'm not sure how meaningful the version is anyway.

Reply:
Yeah I don't think the version of the template matters, other than that the guidelines rule complains if the major version is zero.